src/pybind/rados/rados.pyx

```diff
@@ -1277,6 +1278,41 @@ Rados object in state %s." % self.state)
         if ret < 0:
             raise make_ex(ret, "error blacklisting client '%s'" % client_address)
 
+    def map_object(self, obj_name, pool_id=-1):
+        """
+        Map an object
+
+        :param obj_name: the name of the object to find
+        :type obj_name: str
+        :param pool_id: id of the pool to look up
+        :type pool_id: int
+
+        :raises: :class:`Error`
+        :returns: dict of OSD up/acting set
```

Comment:
should specify the type of the contents of the dict, e.g. dict containing 'up' and 'acting' mapped to lists of ints